src/index.ts

```diff
@@ -10,6 +10,14 @@ import { readDefinitionFile } from "@bc-cr/reader";
 
 import { getMappedTarget, getMapping } from "@bc-cr/util/mapping";
 
+import { ArchiveArtifacts } from "@bc-cr/domain/archive-artifacts";
+import { Command, CommandLevel, BuildCommand, Build } from "@bc-cr/domain/build";
+import { DefinitionFile } from "@bc-cr/domain/definition-file";
+import { Dependency } from "@bc-cr/domain/dependencies";
+import { Mapping } from "@bc-cr/domain/mapping";
+import { Node } from "@bc-cr/domain/node";
+import { ReaderOpts } from "@bc-cr/domain/readerOptions";
```

Comment:
what is
```suggestion
import * as Domain from "@bc-cr/domain/";
```
 same strategy should be applied for the rest of domain submodules
and then 

```
export {
...
...
Domain
};
```

Reply:
We will end up exporting all the domain objects even the ones which are mainly for internal use of the package. Is that okay?

Reply:
I prefer the 1st option in that case. Will make the change

Reply:
Nvm typescript complains about this line:
```typescript
import * as Domain from "@bc-cr/domain/";
```